src/status_im2/contexts/chat/messages/contact_requests/bottom_drawer.cljs

```diff
@@ -0,0 +1,26 @@
+(ns status-im2.contexts.chat.messages.contact-requests.bottom-drawer
+  (:require
+   [i18n.i18n :as i18n]
+   [utils.re-frame :as re-frame]
+   [quo2.core :as quo]
+   [quo2.components.drawers.permission-context.view :as permission-context]
+   [status-im.constants :as constants]))
```

Comment:
We have constant files in `status-im2.common.constants` namespace, probably we should be using that one and deprecate the constant namespace from `status-im`